aten/src/ATen/native/cuda/Repeat.cu

```diff
@@ -17,7 +17,13 @@ __global__ static void compute_cuda_kernel(
     index_t* result_ptr,
     int64_t size,
     int64_t result_size) {
-  CUDA_KERNEL_ASSERT(result_size == cumsum_ptr[size - 1]);
+  if (C10_UNLIKELY((result_size != cumsum_ptr[size - 1]))) {
+    printf("%s:%d:%s: block: [%d,%d,%d], thread: [%d,%d,%d] "
+      "Invalid input! In `repeat_interleave`, the `output_size` argument (%ld) must be the same as the sum of the elements in the `repeats` tensor (%ld).\n",
+      __FILE__, __LINE__, __func__,blockIdx.x, blockIdx.y, blockIdx.z, threadIdx.x, threadIdx.y, threadIdx.z, result_size, cumsum_ptr[size - 1 ]);
+    CUDA_KERNEL_ASSERT(result_size == cumsum_ptr[size - 1])
```

Comment:
```suggestion
    CUDA_KERNEL_ASSERT_MSG(result_size == cumsum_ptr[size - 1]), fmt::printf("%s:%d:%s: block: [%d,%d,%d], thread: [%d,%d,%d] "
      "Invalid input! In `repeat_interleave`, the `output_size` argument (%ld) must be the same as the sum of the elements in the `repeats` tensor (%ld).\n",
      __FILE__, __LINE__, __func__,blockIdx.x, blockIdx.y, blockIdx.z, threadIdx.x, threadIdx.y, threadIdx.z, result_size, cumsum_ptr[size - 1 ]);
```
A bit cleaner using fmtlib here

Reply:
Much nicer, let me try this out, thank you!

Reply:
Ah... :( https://github.com/pytorch/pytorch/blob/7caf6c801ddfaf556a3ca191173b50002c4261f4/c10/macros/Macros.h#L372